README.md

```diff
@@ -17,43 +17,30 @@ and start making your graphql requests against:
 ```
 
 # Features
-## Filtering
-You can filter any doctype by its `name`, `standard_filters` or with a `filters` json
-
-Filtering by name is straight forward:
-```graphql
+## Getting single Document and getting a filtered doctype list
+You can get a single document by its name using `<doctype>` query.  
+```
 {
     User(name: "Administrator") {
         name,
-        email,
-        roles {
-            role
-        }
+        email
     }
 }
 ```
-For standard filters, lets take a look at the User doctype. It has 3 standard filters:
-- `full_name`
-- `username`
-- `user_type`
-
-It is possible to query them like this:
+You can get a list of documents by querying `<doctype-plural>`. You can also pass in filters and sorting details as arguments:
 ```graphql
 {
-    User(full_name: "Test A") {
-        name, email
+    Users(filter: [["name", "like", "%a%"]], sortBy: { field: NAME, direction: ASC }) {
+        name,
```

Comment:
@faztp12 I think this should be updated for the connection structure when it's a plural

Reply:
@assemmarwan updated 👍